packages/@tinacms/cli/src/cmds/init/static.ts

```diff
@@ -0,0 +1,269 @@
+/**
+Copyright 2021 Forestry.io Holdings, Inc.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+import path from 'path'
+import { logText, successText } from '../../utils/theme'
+import { logger } from '../../logger'
+import fs from 'fs-extra'
+import prompts from 'prompts'
+import { execShellCommand } from '.'
+import { Telemetry } from '@tinacms/metrics'
+
+export async function initStaticTina(ctx: any, next: () => void, options) {
+  logger.level = 'info'
+  // Choose package manager
+  const packageManager = await choosePackageManager()
+
+  // Choose typescript
+  const usingTypescript = await chooseTypescript()
+
+  // Choose public folder
+  const publicFolder = await choosePublicFolder()
+
+  // Report telemetry
+  await reportTelemetry(usingTypescript, options.noTelemetry)
+
+  // Check for package.json
+  const hasPackageJSON = await fs.pathExistsSync('package.json')
+  // if no package.json, init
+  if (!hasPackageJSON) {
+    await createPackageJSON()
+  }
+
+  // Check if .gitignore exists
+  const hasGitignore = await fs.pathExistsSync('.gitignore')
+  // if no .gitignore, create one
+  if (!hasGitignore) {
+    await createGitignore()
+  } else {
+    const hasNodeModulesIgnored = await checkGitignoreForNodeModules()
+    if (!hasNodeModulesIgnored) {
+      await addNodeModulesToGitignore()
+    }
+  }
+
+  await addDependencies(packageManager)
+
+  await addConfigFile(publicFolder, usingTypescript)
+
+  await addContentFile()
+
+  logNextSteps(packageManager)
+}
+const baseDir = process.cwd()
+
+const choosePackageManager = async () => {
```

Comment:
Might be out of scope but it might be nice to eventually add pnpm to the normal init as well.

Reply:
I'm hoping we can just start moving pieces of the normal init into this one as standalone becomes the happy path. For example, the main difference between the two now (aside from `defineStaticConfig`) is the stuff for NextJS, which feels like it could be simple to add